scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -158,6 +160,28 @@ workflow Mutect2 {
 
 
   if (is_run_oncotator) {
+        call GetSampleName as GetTumorSample {
```

Comment:
Why didn't you just output the sample names from the M2 task?  Wasn't that what was happening before?

Reply:
No action required other than to answer the question.  Though I'd be grateful if we can avoid having extra localizing of bams.

Reply:
I threw the baby out with the bathwater.  I will restore the baby.